fps-analysis/report-fps-traces.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env python3
+
+import re
+import argparse
+import math
+import statistics
+import csv
+from scipy import stats
+import numpy as np
+import pathlib
+import sys
+
+def format_percent(frac):
+    """Converts a factional value (typically 0.0 to 1.0) to a string as a percentage"""
+    if abs(frac) > 0.0 and abs(frac) < 0.0001:
+        return "<.01%"
```

Comment:
code and string don't have the same epsilon value

Reply:
Really? It is 0.1%, so it is 0.1 * 1/100 = 0.0001

FWIW, this was C&P directly from shader-db report.py. If in the end we agree that it is an error, we could notify them too.